tests/workflows/missing_node_output/workflow.py

```diff
@@ -2,6 +2,7 @@
 
 from vellum.workflows.nodes.bases import BaseNode
 from vellum.workflows.outputs.base import BaseOutput
+from vellum.workflows.types import MergeBehavior
```

Comment:
[nit] I'd remove it from this test to keep the test simpler

Reply:
oh jk we need it bc the attribute is missing and that's what the test is focused on. In that case, I'd remove from StartNode and change NextNode to an AWAIT_ALL